### PR TITLE
Sorted the country codes from least to greatest

### PR DESCRIPTION
**Pull request by Kyle-Ski**
Sorted the country codes by numerical value for Kazakhstan and US

### CODE REVIEW

**Comment by bl00mber** [comment]
Thanks for the PR! What's the point of this change?

**Comment by Kyle-Ski** [comment]
The other countries with local codes had them sorted numerically already, so I sorted the rest to make it consistent

File: src/country_data.js [hunk 1]
```diff
@@ -681,7 +681,7 @@ const rawAllCountries = [
     'kz',
     '7',
     '+. ... ...-..-..',
-    1, ['313', '327', '7172', '312', '73622', '321', '324', '336', '318', '315', '325', '311', '326', '310']
+    1, ["310", "311", "312", "313", "315", "318", "321", "324", "325", "326", "327", "336", "7172", "73622"]
```
Comment (bl00mber) [inline src/country_data.js]: Why you've replaced single quotes to double? There is only single quotes used across the file.

**Comment by bl00mber** [comment]
Closed due to inactivity.

**Pull request closed without merging**